robottelo/ui/base.py

```diff
@@ -429,6 +429,9 @@ def auto_complete_search(self, go_to_page, entity_locator, partial_name,
         self.wait_for_ajax()
         strategy, value = common_locators['auto_search']
         self.click((strategy, value % name))
+        # Intentionally clicking second time because sometimes first time
+        # click just selects/highlights the item but not clicking it
+        self.click((strategy, value % name))
```

Comment:
Can you please try if using self.click(<locator>, scroll=True) works here ?

This could be related to https://github.com/SatelliteQE/robottelo/pull/3091

The above approach helped me reliably get past `repo enable` failures. Currently I feel any issues with click may require this fix. 

This fix is not required by default as it's only required when the element is not into the visible area of
the browser window.